src/server/exemption/site-details/delete-site/index.njk

```diff
@@ -0,0 +1,48 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "cancel-button/macro.njk" import appCancelButton %}
+
+{% block beforeContent %}
+  <nav>
+    {{ govukBackLink({
+      text: "Back",
+      href: backLink
+    }) }}
+  </nav>
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">{{heading}}</h1>
+
+      {{ govukInsetText({
+        text: 'Site ' + siteNumber
+      }) }}
+
+      <form method="POST">
+        <input type="hidden" name="csrfToken" value="{{csrfToken}}">
+        <input type="hidden" name="siteIndex" value="{{siteIndex}}">
+
+        <div class="govuk-button-group">
+          {{ govukButton({
```

Comment:
Just in case the network connection is slow, might be worth adding `data-prevent-double-click` to the button so the user can't click it twice - https://design-system.service.gov.uk/components/button/#stop-users-from-accidentally-sending-information-more-than-once